src/clint.py

```diff
@@ -899,7 +899,8 @@ def CheckIncludes(filename, lines, error):
                     not name.endswith('_defs.h') and
                     not name.endswith('h.inline.generated.h') and
                     not name.endswith('_defs.generated.h') and
-                    not name.endswith('_enum.generated.h')):
+                    not name.endswith('_enum.generated.h') and
+                    not name.endswith('_vars.generated.h')):
```

Comment:
`option_vars.generated.h` not equivalent to a `_defs` header, as it contains `EXTERN` variables.